qiskit/pulse/library/waveform.py

```diff
@@ -25,6 +25,9 @@ class Waveform(Pulse):
     """A pulse specified completely by complex-valued samples; each sample is played for the
     duration of the backend cycle-time, dt.
     """
+
+    limit_amplitude = False
+
     def __init__(self, samples: Union[np.ndarray, List[complex]],
                  name: Optional[str] = None,
                  epsilon: float = 1e-7):
```

Comment:
I would prefer to add this as a keyword argument with a default value of `True` (I understand it is this way likely just for a quick test).

Reply:
@taalexander 

* Added kwarg for `limit_amplitude` with default value `None` so that the class attribute is used
* I placed `limi_amplitude` before the `epsilon` in the arguments list.
* Default value for `limit_amplitude` is set to `True`, which is consistent with the old behavior.
* One could combine `limit_amplitude` and `epsilon` this into a single variable and let `epsilon=None` skip the amplitude check, but I prefer the current approach.